tests/utils/resources.py

```diff
@@ -11,19 +12,24 @@
     'resnet50': 'https://github.com/onnx/models/raw/main/vision/classification/resnet/model/resnet50-v2-7.onnx',
 }
 
+_BASE_URL = 'https://gitlab.expasoft.com/p.ivanov/onnx2torch_data/-/raw/main/models_for_tests'
+
+_CHKP_DETECTION_URL = f'{_BASE_URL}/detection'
+_CHKP_SEGMENTATION_URL = f'{_BASE_URL}/segmentation'
+_CHKP_TRANSFORMERS_URL = f'{_BASE_URL}/transformers'
+
 _ONNX_MODELS_IDS = {
```

Comment:
1) move _ONNX_MODELS_URLS to _ONNX_MODELS_IDS
2) rename _ONNX_MODELS_IDS -> _ONNX_MODELS_URLS